projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -426,4 +428,27 @@ public function remove_submenus() {
 			$_registered_pages[ 'admin_page_' . $page_slug ] = true; // phpcs:ignore
 		}
 	}
+
+	/**
+	 * Add the calypso /woocommerce-installation/ menu item.
+	 */
+	public function add_woocommerce_installation_menu() {
+		/**
+		 * Whether to show the WordPress.com WooCommerce Installation menu.
+		 *
+		 * @use add_filter( 'jetpack_show_wpcom_woocommerce_installation_menu', '__return_true' );
+		 * @module masterbar
+		 * @since 9.3.3
```

Comment:
That would be 10.3.0 now.

```suggestion
		 * @since 10.3.0
```